pkg/configure/cmd.go

```diff
@@ -0,0 +1,65 @@
+package configure
+
+import (
+	"path/filepath"
+
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/container"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/pod"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/config_files"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const (
+	configDirFlag = "config-directory"
+)
+
+var (
+	configDirectory string
+)
+
+func AddFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&configDirectory, configDirFlag, "", "(Optional) Path where enrichment/configuration files will be written.")
+
+	container.AddFlags(cmd)
+	pod.AddFlags(cmd)
+
+}
+
+func Execute(fs afero.Afero, to string) error {
+	if configDirectory == "" {
+		return nil
+	}
+
+	path := filepath.Join(to, configDirectory)
```

Comment:
```suggestion
```

You want to create the config files under `<config-dir>`
and not under `<target>/<config-dir>`

the `to` part you do not need (I will need it for the processmoduleconfig)

Reply:
done https://github.com/Dynatrace/dynatrace-bootstrapper/pull/8/commits/8b881abb6554665793f20bbd60eb113caa65f219